common/client.go

```diff
@@ -53,6 +53,7 @@ type DatabricksClient struct {
 	ConfigFile string `name:"config_file" env:"DATABRICKS_CONFIG_FILE"`
 
 	GoogleServiceAccount string `name:"google_service_account" env:"DATABRICKS_GOOGLE_SERVICE_ACCOUNT" auth:"google"`
+	GoogleCredentials    string `name:"google_credentials" env:"DATABRICKS_GOOGLE_CREDENTIALS" auth:"google,sensitive"`
```

Comment:
Okay, so we need to add GOOGLE_CREDENTIALS and GOOGLE_APPLICATION_CREDENTIALS properties in order to keep semantics reusable from configured GCP provider: https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference

I'll give it another read in few hrs

Reply:
Done. But it may be tricky if user wants to configure different credentials for Google Terraform and Databricks Terraform providers through the env variable for some reason.

Reply:
that's intentional and by design :) default behavior - reuse service principals for the same pipeline, custom behavior goes through provider attributes